assets/js/bootstrap-migration/common-components/login-form.tpl.liquid

```diff
@@ -0,0 +1,45 @@
+<form accept-charset="UTF-8" action="{% raw %}{{ $ctrl.loginUrl }}{% endraw %}" method="post" id="customer_login" name="customer_login">
+    {% include 'bootstrap-migration/form-errors-custom' %}
+    <fieldset>
+        <div class="row">
+            <div class="col-md-12">
+                <div class="form-group mt-0">
+                    <input class="form-control" ng-model="$ctrl.username" name="customer[user_name]" required placeholder="{{ 'customer.sign_in.user_name' | t }}" />
+                    <span class="text-danger" ng-show="!customer_login['customer[user_name]'].$pristine && customer_login['customer[user_name]'].$error.required">{{ 'common.required' | t }}</span>
+                </div>
+            </div>
+            <div class="col-md-12">
+                <div class="mb-2">
+                    <input class="form-control" ng-model="$ctrl.password" type="password" name="customer[password]" required placeholder="{{ 'customer.sign_in.password' | t }}" />
+                    <span class="text-danger" ng-show="!customer_login['customer[password]'].$pristine && customer_login['customer[password]'].$error.required">{{ 'common.required' | t }}</span>
+                </div>
+            </div>
+        </div>
+
+        {% if template != 'customers/login' %}
+            <div class="mb-2">
+                {% include 'bootstrap-migration/customers/forgot-links' %}
+            </div>
+        {% endif %}
+
+        {% if template != 'customers/login' %}
+            <div class="row">
+                <div class="col-6 col-md-5">
+                    <input type="submit" class="btn btn-primary btn-block" value="{{ 'customer.sign_in.sign_in' | t }}">
+                </div>
+                <div class="col-6 col-md-5">
+                    {{ 'layout.customer.register' | t | customer_register_link | replace: 'a href', 'a class="btn btn-outline-primary btn-block" href' }}
+                </div>
+            </div>
+        {% else %}
+            <div class="form-group mb-5">
+                {% include 'bootstrap-migration/customers/forgot-links' %}
+            </div>
+            <div>
+                <input type="submit" class="btn btn-primary w-50" value="{{ 'customer.sign_in.continue' | t }}">
+            </div>
+        {% endif %}
+    </fieldset>
+    <input name="RememberMe" type="hidden" value="true" />
```

Comment:
Move to localization

Reply:
That's not display name, it's `input` name, i.e. field in model in storefront. So don't need to fix anything